databases/week2/week2_part1.sql

```diff
@@ -0,0 +1,30 @@
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+insert into task(title, description, created, updated, due_date, status_id, user_id)
+values('meeeting','meeting with comapany owner','2021-01-15 task','2021-01-19 06:54:16','2021-02-18 06:54:16',2,3);
```

Comment:
```suggestion
values('meeeting','meeting with comapany owner','**2021-01-15 task**','2021-01-19 06:54:16','2021-02-18 06:54:16',2,3);
```
I think 'created' as a DATE datatype, Its not allow to insert string.

Reply:
you can use NOW(),
https://www.w3schools.com/sql/func_mysql_now.asp